scss/_alerts.scss

```diff
@@ -1,17 +1,18 @@
 .flash-messages {
-  margin-top: 15px;
-  margin-bottom: 15px;
+  margin-top: 16px;
```

Comment:
Interesting idea. Personally I think that since the `margin` shorthand property is ordered `top`, `right`, `bottom`, `left`, that should be preserved even if splitting those margins into separate declarations. Plus it's a natural way to look around an object; my eyes don't look around like `bottom`, `left`, `right`, then `top`.

@mdo Thoughts?